### PR TITLE
Add types

### LINKED ISSUE #5: Typescript support
### Subject of the feature
This package lacks to **Typescript** support despite `react-markdown` already offers support for it,

### Problem

We are trying to migrate our current portffolio from **Javascript** to **Typscript** component by component and we found the next issue related to this package:

```shell
<html>TS7016: Could not find a declaration file for module 'remark-unwrap-images'. '/home/luisalaguna/Projects/portfolio/frontend2.0/node_modules/remark-unwrap-images/index.js' implicitly has an 'any' type.<br/>Try `npm i --save-dev @types/remark-unwrap-images` if it exists or add a new declaration (.d.ts) file containing `declare module 'remark-unwrap-images';`
```

### Expected behavior

It should have a minimal typscript support.

### Alternatives

As this is a very specifical plugin, it has no other alternative.

### PR DESCRIPTION
**Pull request by SalahAdDin**
It fixes https://github.com/remarkjs/remark-unwrap-images/issues/5.

### CODE REVIEW

File: types/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "compilerOptions": {
+      "lib": ["es2015"],
+      "strict": true,
+      "baseUrl": ".",
+      "paths": {
+        "remark-unwrap-images": ["."]
+      }
+    }
+  }
```
Comment (ChristianMurphy) [inline types/tsconfig.json]: could prettier be run on this file, to align indentation and add the missing line feed at the end of file?

File: types/tslint.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "dtslint/dtslint.json",
+  "rules": {
+    "semicolon": false,
+    "whitespace": false
+  }
+}
```
Comment (ChristianMurphy) [inline types/tslint.json]: could prettier be run on this file, to add the missing line feed at the end of file?

File: package.json [hunk 3]
```diff
@@ -44,10 +47,11 @@
     "xo": "^0.37.0"
   },
   "scripts": {
-    "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
+    "format": "remark . -qfo --ignore-pattern test/ && prettier . -w --loglevel warn && xo --fix --ignore types",
```
Comment (ChristianMurphy) [inline package.json]: the test folder wasn't modified, it shouldn't need to be ignored
```suggestion
    "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix --ignore types",
```

**Review by ChristianMurphy — APPROVED** [review]
Thanks @SalahAdDin!
A couple minor tweaks and this should be good to go

**Comment by SalahAdDin** [comment]
@ChristianMurphy done.

**Comment by ChristianMurphy** [comment]
Thanks @SalahAdDin! :bow:

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
thanks, released!

**Pull request merged**